opal/mca/event/libevent2022/libevent/evthread-internal.h

```diff
@@ -47,55 +47,55 @@ struct event_base;
 #if ! defined(_EVENT_DISABLE_THREAD_SUPPORT) && defined(EVTHREAD_EXPOSE_STRUCTS)
 /* Global function pointers to lock-related functions. NULL if locking isn't
    enabled. */
-extern struct evthread_lock_callbacks _evthread_lock_fns;
-extern struct evthread_condition_callbacks _evthread_cond_fns;
-extern unsigned long (*_evthread_id_fn)(void);
-extern int _evthread_lock_debugging_enabled;
+extern struct evthread_lock_callbacks opal_evthread_lock_fns;
+extern struct evthread_condition_callbacks opal_evthread_cond_fns;
+extern unsigned long (*opal_evthread_id_fn)(void);
+extern int opal_evthread_lock_debugging_enabled;
```

Comment:
You should probably put all libevent-based renamings in `opal/mca/event/libevent2022/libevent/opal_rename.h`.